docs/new-contributors.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
+*   **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able to add basic virtualization functionality to Kubernetes.
+
+Even that KubeVirt is an add-on to Kubernetes, both of them have things in common:
+
+*   Mostly written in golang
+*   Often related to distributed microservice architectures
+*   Declarative and Reactive (Operator pattern) approach
+
+This short page shall help to get started with the projects and topics surrounding them
+
+## Projects & Communities
+
+### [Kubernetes](http://kubernetes.io/) 
+
+*   Getting started
+    *   [http://kubernetesbyexample.com](http://kubernetesbyexample.com)
+    *   [Hello Minikube - Kubernetes](https://kubernetes.io/docs/tutorials/stateless-application/hello-minikube/ "https://kubernetes.io/docs/tutorials/stateless-application/hello-minikube/") 
+    *   [User Guide - Kubernetes](https://kubernetes.io/docs/user-guide/ "https://kubernetes.io/docs/user-guide/") 
+*   Details
+    *   [Declarative Management of Kubernetes Objects Using Configuration Files - Kubernetes](https://kubernetes.io/docs/concepts/tools/kubectl/object-management-using-declarative-config/ "https://kubernetes.io/docs/concepts/tools/kubectl/object-management-using-declarative-config/")
+    *   [Kubernetes Architecture](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture.md)
+
+### [KubeVirt](https://github.com/kubevirt/) 
+
+*   Getting started  
+    *   [Demo](https://github.com/kubevirt/demo)
+*   Details
+
+## Topics
+
+### golang
+
+*   Getting started  
+
+    *   [Documentation - The Go Programming Language](https://golang.org/doc/ "https://golang.org/doc/") 
+    *   [Getting Started - The Go Programming Language](https://golang.org/doc/install "https://golang.org/doc/install") 
+
+Patterns
+
+*   [Introducing Operators: Putting Operational Knowledge into Software](https://coreos.com/blog/introducing-operators.html "https://coreos.com/blog/introducing-operators.html") 
+*   [Microservices](https://martinfowler.com/articles/microservices.html "https://martinfowler.com/articles/microservices.html") nice content by Martin Fowler
```

Comment:
Formatting in this block looks kind of odd.   'Getting started' is in a list bullet, but 'Patterns' is not ?!?